src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/ImportOrderCheck.java

```diff
@@ -158,16 +185,36 @@
  * </li>
  * </ul>
  * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * import static java.lang.System.out; //Violation; it should be 'sorted'
+ * import static java.lang.Math.*; //OK
+ * import java.io.IOException; //Violation; Groups aren't separated by blank line
+ *
+ * import java.net.URL; //OK
+ * import java.security.KeyManagementException; //OK
+ *
+ * import javax.net.ssl.TrustManager; //OK
+ *
+ * import javax.net.ssl.X509TrustManager; //Violation; Groups should not separate internally
+ *
+ * import org.apache.http.conn.ClientConnectionManager; //OK
+ * import org.apache.http.conn.scheme.Scheme; //OK
+ *
+ * public class SomeClass { ... }
+ * </pre>
+ * <p>
```

Comment:
This example should follow the configuration on lines 221-226.

Reply:
sorry i didnt get this.
are static group and java group should not be separated?